build.gradle.kts

```diff
@@ -56,6 +57,9 @@ if (project.version == "unspecified") {
     edcVersion = project.version as String
 }
 
+// required by the nexus publishing plugin
+project.version = edcVersion
```

Comment:
at this point we could get rid of the `edcVersion` variable and use only `project.version`. same for the `if` statement at line 51